test/feature-benchmark/mzcompose.py

```diff
@@ -46,6 +37,14 @@
 from materialize.feature_benchmark.executor import Docker
 from materialize.feature_benchmark.filter import Filter, FilterFirst, NoFilter
 from materialize.feature_benchmark.measurement import MeasurementType
+from materialize.feature_benchmark.scenarios.scenarios import *  # noqa: F401 F403
+from materialize.feature_benchmark.scenarios.scenarios import MySqlStreaming, Scenario
+from materialize.feature_benchmark.scenarios.scenarios_concurrency import *  # noqa: F401 F403
```

Comment:
Do we still need the scenarios_ in the filename? Why not 
```suggestion
from materialize.feature_benchmark.scenarios.concurrency import *  # noqa: F401 F403
```

Reply:
Yeah, makes sense. Done that. I had to rename `scenarios` to `benchmark_main` to have it unique (due to the doctests).